### ISSUE TITLE
1001tracklists

### ISSUE THREAD

**Issue by Rimole** [ineshbose/simple-icons-latex#19]
```
\documentclass{article}
\usepackage{simpleicons}
\begin{document}
\simpleicon{1001tracklists}
\end{document}
```

Compiling with XeLaTeX or LuaLaTeX: While the icon is printed, the message shows:

Missing character: There is no  (U+EA01) in font SimpleIcons Filtered/OT:language=dflt;!

Compiling with pdfLaTeX: The icon is not printed and the message shows: 

Missing character: There is no ^^@ in font SimpleIcons--simpleiconsOne!

**Comment by hair-splitter** [comment]
The problem is the same with pdflatex.

**Comment by Rimole** [comment]
When using fontforge to look at the SimpleIcons.otf, the glyph EA02 is named /1password (and that symbol shows in the pdf), while gylph EA01 is named /.notdef instead of /1001tracklists and, naturally, that glyph does not show in the pdf - this glyph needs to be renamed!

**Comment by ineshbose** [comment]
Thanks - sorry it took a while to get to this. I realised that an OTF needs a `.notdef` glyph which is usually the first one and this caused conflicts. Icon should be available in v9.19.0 (confirm with manual when reflected in all mirrors).
